SYCL/ESIMD/Stencil.cpp

```diff
@@ -10,6 +10,8 @@
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: %HOST_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// Temporary disabled on Windows
```

Comment:
Please provide some justification in the commit message.

Reply:
@DenisBakhvalov, could you please add comment reqested by Konst?